src/test/java/uk/gov/justice/digital/hmpps/keyworker/utils/JwtAuthenticationHelper.java

```diff
@@ -62,4 +66,33 @@ public String createJwt(final JwtParameters parameters) {
         private List<String> roles;
         private Duration expiryTime;
     }
+
+
+    private KeyPair getKeyPair(final Resource resource, final String alias, final char[] password) {
+        InputStream inputStream = null;
+        KeyStore store;
+        try {
+            store = KeyStore.getInstance("jks");
+            inputStream = resource.getInputStream();
```

Comment:
Could use a try-with-resources statement on this inputstream to avoid the need for the finally block?